src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -126,24 +126,20 @@ public static void setFormatsToRepoConfigs(List<RepoConfiguration> configs, List
      * Clears authors information and use the information provided from {@code standaloneConfig}.
      */
     public void update(StandaloneConfig standaloneConfig) {
-        authorList.clear();
+        authorList = new ArrayList<>();
```

Comment:
unnecessary change?

Reply:
this is a change necessary for me to use GitTestTemplate as it initializes authorList to be non-mutable by default.